killbill/__init__.py

```diff
@@ -1,42 +1,114 @@
-#
-# Copyright 2011-2014 Ning, Inc.
-# Copyright 2014-2017 Groupon, Inc.
-# Copyright 2014-2017 The Billing Project, LLC
-#
-# The Billing Project, LLC licenses this file to you under the Apache License, version 2.0
-# (the "License"); you may not use this file except in compliance with the
-# License.  You may obtain a copy of the License at:
-#
-#    http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
-# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
-# License for the specific language governing permissions and limitations
-# under the License.
-#
-
-base_uri = 'http://localhost:8080'
-username = 'admin'
-password = 'password'
-api_key = 'bob'
-api_secret = 'lazar'
-
-try:
```

Comment:
Note that all of that stuff will be important to support Python 2.5, and to support custom HTTP backends (see details in the ticket).